SolutionAudit/AuditSolution.cs

```diff
@@ -44,9 +43,10 @@ public AuditSolution(String solutionPath, Options options)
 
             Log.Debug("Auditing projects");
 
-            Projects = Solution.Load(solutionPath)
-                .Projects
-                .AsParallel()
```

Comment:
why not parallel?

Reply:
This was removed in your commit, some sanity testing showed there might be issues with how GlobalProjectCollection loads projects depending on the flags you provide. I added some locks in key areas and those errors went away and we  get the AsParallel benefit again. which cuts the time at least in half I've found.